10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>The rest of the sections need some explanation to describe what the potential is, how it affects the simulation, and when a user might want to add it (or not).</p><p> </p><p>For a more visual guide, you might want to run the simulation without any rigidity potentials first to show the user the flexible mesh. Then turn on the rigidity potentials (possibly one at a time) to show the resulting behavior.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>